src/modules/ecdh/tests_impl.h

```diff
@@ -142,11 +150,53 @@ static void test_result_basepoint(void) {
     }
 }
 
+static void test_ecdh_wycheproof(void) {
+#include "../../wycheproof/ecdh_secp256k1_test.h"
+    int t;
+    for (t = 0; t < SECP256K1_ECDH_WYCHEPROOF_NUMBER_TESTVECTORS; t++) {
+        int parsed_ok;
+        secp256k1_pubkey point;
+        const unsigned char *pk;
+        const unsigned char *sk;
+        const unsigned char *expected_shared_secret;
+        unsigned char output_ecdh[65] = { 0 };
+
+        int expected_result;
```

Comment:
Can we rename this to `expected_parses` or similar? `result` is a bit imprecise.